Finished the project

kjanos89 opened the pull request.

Doc415 reviewed (changes requested):
Hey when i try to update email of a record, it does not update. It is still the old one when i display records.
Phone number validation should be handled better. It takes 2 and 22222222222222222222222222222222 as valid phone number.
You can use Regex or there are couple of validation libraries you can use.
Waiting for your changes.

kjanos89 commented:
Hey there, @Doc415 ! Thanks for the review. I've updated my phone number validation to be a bit more strict upon phone number inputs, minimum length is now 3 (so that people can save the local help lines for example as direct contacts) and maximum length is 15 as a quick google search told me that is the accepted maximum length. 
Also updated the contact updating method, these will now prompt the user when the input is wrong and will let the user restart from the beginning. Updating is a bit more important so instead of letting the user redo the same step I just reset the whole updating process to make sure the updated information will be okay. The steps that do not require any update can still be left empty to be ignored when the update occurs.

Doc415 reviewed (approved):
Thanks for the changes. Approving now.
Good luck with the next projects.
